spinalcordtoolbox/deepseg/models.py

```diff
@@ -77,6 +78,24 @@
         "contrasts": ["t2"],
         "default": False,
     },
+    "model_seg_ms_sc_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_sc_mp2rage.zip"
+        ],
+        "description": "Segmentation of spinal cord on MP2RAGE data from MS participants.",
+        "contrasts": ["mp2rage"],
+        "default": False,
+    },
+    "model_seg_ms_lesion_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_lesion_mp2rage.zip"
+        ],
+        "description": "Segmentation of multiple sclerosis lesions on cropped MP2RAGE spinal cord data. To crop the "
+                       "data you can first segment the spinal cord using the model 'model_seg_ms_lesion_mp2rage' and "
+                       "then crop the MP2RAGE image using 'sct_crop_image -i IMAGE -m IMAGE_seg'",
```

Comment:
More broadly, I'm realizing that this description isn't actually seen by users, is it?  (We have `-list-tasks`, but that will only show the short description under `task`.) This problem will be somewhat handled by #3541, however, so I'll make sure to update that PR with what you've written here. 

(And, for now we can make the usage clear in whatever materials/tutorials are sent to our collaborators.)

Reply:
> This problem will be somewhat handled by #3541, however, so I'll make sure to update that PR with what you've written here.

indeed, that was my thinking. Let's start documenting better each model, anticipating that at some point this description will be exposed.